chore: add trouble shoot for newrelic maui agent (#19944)
newrelic/docs-website: src/content/docs/mobile-monitoring/new-relic-mobile-maui-dotnet/monitor-your-net-maui-application.mdx

Missing HTTP data in the UI?

After installing the .NET MAUI agent, wait at least 5 minutes. If no HTTP data appears on the HTTP errors and HTTP requests UI pages, make sure you used `HttpMessageHandler` in `HttpClient`.

### Missing Distributed Tracing Data in the UI

> If you're experiencing issues with missing Distributed Tracing data in the UI and are using static methods to report HTTP data, please note that Distributed Tracing will not function under these circumstances. To enable Distributed Tracing, ensure that you are using `HttpMessageHandler` within `HttpClient`.

Reviewer: ```suggestion
Distributed Tracing does not work when you use static methods to report HTTP data. To enable Distributed Tracing, you must use `HttpMessageHandler` with `HttpClient`.
```
